dev/fattest.simplicity/src/componenttest/topology/impl/LibertyServer.java

```diff
@@ -858,16 +858,40 @@ public void reconfigureServerUsingExpandedConfiguration(String testName, String
      */
     public void reconfigureServerUsingExpandedConfiguration(String testName, String configDir, String newConfig, boolean resetMark, String... waitForMessages) throws Exception {
 
+        String thisMethod = "reconfigureServerUsingExpandedConfiguration";
         ServerFileUtils serverFileUtils = new ServerFileUtils();
         String newServerCfg = serverFileUtils.expandAndBackupCfgFile(this, configDir + "/" + newConfig, testName);
-        Log.info(c, "reconfigureServerUsingExpandedConfiguration", "Reconfiguring server to use new config: " + newConfig);
+        Log.info(c, thisMethod, "Reconfiguring server to use new config: " + newConfig);
         if (resetMark) {
             setMarkToEndOfLog();
         }
         replaceServerConfiguration(newServerCfg);
 
         Thread.sleep(200); // Sleep for 200ms to ensure we do not process the file "too quickly" by a subsequent call
         waitForConfigUpdateInLogUsingMark(listAllInstalledAppsForValidation(), waitForMessages);
+
+        // wait for ssl port restart
+        waitForSSLRestart();
+
+    }
+
+    public void waitForSSLRestart() throws Exception {
+
+        String thisMethod = "waitForSSLRestart";
+        // look for the "CWWKO0220I: TCP Channel defaultHttpEndpoint-ssl has stopped listening for requests on host " message
+        // if we find it, then wait for "CWWKO0219I: TCP Channel defaultHttpEndpoint-ssl has been started and is now listening for requests on host"
+        String sslStopMsg = waitForStringInLogUsingMark("CWWKO0220I: TCP Channel defaultHttpEndpoint-ssl has stopped listening", 500);
+        if (sslStopMsg != null) {
+            String sslStartMsg = waitForStringInLogUsingMark("CWWKO0219I: TCP Channel defaultHttpEndpoint-ssl has been started and is now listening");
+            if (sslStartMsg == null) {
+                Log.info(c, thisMethod, "SSL may not have started properly - future failures may be due to this");
```

Comment:
Would it be any help to make this a `Log.warning()` message?

Reply:
I can make the update - will also need to allow the warning so the log scanner doesn't complain about it during shutdown.